### ISSUE TITLE
song shown on toasty is literally wrong 99% of times

### ISSUE THREAD

**Issue by wtf16** [aleab/toastify#172]
the current song playing toast, when hotkey is pressed, is always stuck on a previous one from AGES ago, and never rectifies itself unless the application was restarted, and this behavior happens every time i use the show current song feature(after the app has been running a while)

**Comment by pedroagp** [comment]
yep, realized this a few days ago as well, probably something changed in the spotify API since this was working without this issue before.
